projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -97,6 +117,10 @@ This can be important for identifying inconsistencies in how classes are labeled
 
 6. Identifying Potentially Overspecified Class Definitions
 
+Title: Overspecified Class Detection
+Constraint Description: Identifies classes with potentially redundant or overlapping property definitions.
+Severity: Warning
+
```

Comment:
Are you trying to find classes with two definitions? You need to use cco:definition somewhere

Reply:
Thanks, I'll change this one :)